index.html

```diff
@@ -1,122 +1,120 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8">
-    <title>Fork Me! FCC: Test Suite Template</title>
-    <link rel="stylesheet" href="css/style.css">
-    <script src="https://kit.fontawesome.com/63ff2cfeab.js" crossorigin="anonymous"></script> 
-  
-  </head>
 
-  <body>
-    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
-    
-     <header id="header">
-       <nav id="navbar">
-         <a href="#">About</a>
-         <a href="#">Work</a>
-         <a href="#">Contact</a>
-       </nav>
-     </header>
-     <div id="welcome-section">
-       <h1>Hey I am Mimic</h1>
-       <h3>a web developer</h3>
-     </div>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+  <script src="https://kit.fontawesome.com/63ff2cfeab.js" crossorigin="anonymous"></script>
 
-    <div id="project-section">
-      <h1>These are some of my projects</h1>
-      <div id="projects">
-      <div class="project-tile" id="project">
-          <a href="#">
-            <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" >
+</head>
+
+<body>
+  <header id="header">
+    <nav id="navbar">
+      <a href="#">About</a>
+      <a href="#">Work</a>
+      <a href="#">Contact</a>
+    </nav>
+  </header>
+  <div id="welcome-section">
+    <h1>Hey I am Mimic</h1>
+    <h3>a web developer</h3>
+  </div>
+
+  <div id="project-section">
+    <h1>These are some of my projects</h1>
+    <div class="projects">
+      <div class="project-tile project">
+        <a href="#">
+          <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="">
```

Comment:
This is an example inline comment. :slightly_smiling_face:

You can see how it's under the main summary above and point to a specific part of the code where you want to suggest changes or just comment on it generally. :wink:

To do this, make sure you go to the "File changes" tab and use the "Start a review" button for your first review comment.